geode-core/src/main/java/org/apache/geode/internal/cache/AbstractBucketRegionQueue.java

```diff
@@ -223,7 +231,12 @@ protected void loadEventsFromTempQueue() {
       // .getBucketToTempQueueMap().get(getId());
       if (tempQueue != null && !tempQueue.isEmpty()) {
         synchronized (tempQueue) {
+          Map<String, Map<Integer, List<Object>>> regionToDispatchedKeysMap =
```

Comment:
This would be better named as something like "regionToDuplicateEventsMap"

Reply:
updated